src/app/backend/replicasetdetail.go

```diff
@@ -157,6 +157,23 @@ func DeleteReplicaSetWithPods(client *client.Client, namespace string, name stri
 	return nil
 }
 
+// Updates Replica Set based on Replica Set specification
+func UpdateReplicaSet(client *client.Client, replicaSetDetail *ReplicaSetDetail) error {
```

Comment:
Can you write a test for this? Verify that update has been called with everything the same except for pods desired.